src/User/User.php

```diff
@@ -128,25 +125,9 @@ public static function boot()
 
         static::deleted(function (User $user) {
             $user->raise(new Deleted($user));
-
-            // Delete all of the posts by the user. Before we delete them
-            // in a big batch query, we will loop through them and raise a
-            // PostWasDeleted event for each post.
-            $posts = $user->posts()->allTypes();
-
-            foreach ($posts->cursor() as $post) {
-                $user->raise(new PostDeleted($post));
-            }
-
-            $posts->delete();
-
-            $user->read()->detach();
-            $user->groups()->detach();
-            $user->accessTokens()->delete();
-            $user->notifications()->delete();
         });
 
-        static::$dispatcher->dispatch(
+        static::$dispatcher->fire(
```

Comment:
Hmm... as far as I can tell, we are using the dispatcher contract here, which only has `dispatch`. So this should be changed back (along with similar changes, if any).

Reply:
Agreed